aloha-core/src/main/scala/com/eharmony/aloha/models/multilabel/package.scala

```diff
@@ -0,0 +1,74 @@
+package com.eharmony.aloha.models
```

Comment:
Most things here should live at a higher level IMO.

Reply:
That's why they are package private.  They are only there to provide named building blocks to make the final type.

Reply:
I wanted to make the definitions as self contained as possible.